### ISSUE TITLE
loosen analyte role to allow cells, virii?

### ISSUE THREAD

**Issue by obi-bot** [obi-ontology/obi#778]
Analyte role is currently defined to inhere is molecular entity or atom. What about cells, virii for which concentration, count, titer or fraction is measured. In those cases it would seem e.g. the cell is analyte and the blood evaluant.

Reported by: alanruttenberg

Original Ticket: [obi/obi-terms/795](https://sourceforge.net/p/obi/obi-terms/795)

**Comment by zhengj2007** [comment]
Discussed on OBI 2017-11-27 call, we make a broad concept of 'analyte role', 'measurand role', which can be defined as "a material entity that the experiment is designed to measure the presence or concentration of. ", and assert 'analyte role' as it subClass.

**Comment by zhengj2007** [comment]
'measurand role' added, asserted 'analyte role' as its child, see commit: https://github.com/obi-ontology/obi/commit/9cc4f64eeb818bd634513bbef9a06766516d9262

**Comment by zhengj2007** [comment]
Need to do after 'measurand role' added:
1. Shall change 'analyte assay' realizes 'analyte role' to 'measurand role'?
2. Need to add 'measurand role' column in assay template, then move 'virus' and 'cell' if any under 'analyte role' column to this column

**Comment by jamesaoverton** [comment]
About 17 rows in the https://github.com/obi-ontology/obi/blob/master/src/ontology/templates/midlevel-assays.tsv file use the 'analyte' column, and some of these should be 'measurand' instead. (This is a smaller number than I thought.) Randi and I will fix these. I don't see any other problems in the assay templates that are already in OBI.

I've added a 'measurand' column to the [OBI assay review](https://docs.google.com/spreadsheets/d/16Jyn-LEXlSOdqAWlPIkaYCoq9SQIJPHeWPojSBEDppA/edit#gid=0) sheet. I moved some values from 'analyte' to 'measurand' when they were clear to me, but someone else should double-check.
